etc/config/cuda.amazon.properties

```diff
@@ -442,8 +442,16 @@ libs.cueigen.versions.337.version=3.3.7
 libs.cueigen.versions.337.path=/opt/compiler-explorer/libs/eigen/v3.3.7
 libs.cueigen.versions.334.version=3.3.4
 libs.cueigen.versions.334.path=/opt/compiler-explorer/libs/eigen/v3.3.4
-libs.thrustcub.name=Thrust+CUB
-libs.thrustcub.description=CUDA collective and parallel algorithms
+libs.cccl.name=CUDA C++ Core libraries
+libs.cccl.description=Fundamental CUDA C++ libraries: Thrust, CUB, libcudacxx
+libs.cccl.versions=trunk:2002000
+libs.cccl.url=https://github.com/NVIDIA/cccl
+libs.cccl.versions.trunk.version=trunk
+libs.cccl.versions.trunk.path=/opt/compiler-explorer/libs/cccl/trunk/thrust:/opt/compiler-explorer/libs/cccl/trunk/cub:/opt/compiler-explorer/libs/cccl/trunk/libcudacxx/include
+libs.cccl.versions.2002000.version=2.2.0
```

Comment:
The ID is unimportant other than needing to be kept the same "forever", but I think `2002000` is confusing for 2.2.0. Or am I missing something?

Reply:
I was wondering the same, I guess it's to pre-empt for possible 2.10.x where 10 is more than 2 and perhaps that it wouldn't sort correctly then? but we have semver ordering, so it should be fine if you just use id 220

Reply:
other than that the paths look good!

Reply:
I updated it to `220`.